nullaway/src/main/java/com/uber/nullaway/handlers/BaseNoOpHandler.java

```diff
@@ -59,6 +62,19 @@ protected BaseNoOpHandler() {
     // We don't allow creating useless handlers, subclass to add real behavior.
   }
 
+  protected static void reportMatch(
+      NullAway analysis, VisitorState state, Tree errorLocTree, String message) {
+    assert analysis != null && state != null;
+    state.reportMatch(
+        analysis
+            .getErrorBuilder()
+            .createErrorDescription(
+                new ErrorMessage(ErrorMessage.MessageTypes.ANNOTATION_VALUE_INVALID, message),
```

Comment:
Why is the `BaseNoOpHandler` specifying the `ANNOTATION_VALUE_INVALID` message type?  I think the message type should be a parameter here